iot/iot-device-registry-infinispan/src/main/java/io/enmasse/iot/registry/infinispan/server/DeviceRegistryAmqpServer.java

```diff
@@ -3,16 +3,19 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-package io.enmasse.iot.registry.infinispan;
+package io.enmasse.iot.registry.infinispan.server;
 
 import org.eclipse.hono.config.ServiceConfigProperties;
 import org.eclipse.hono.service.amqp.AmqpServiceBase;
 import org.eclipse.hono.util.Constants;
+import org.springframework.stereotype.Component;
 
+@Component
 public final class DeviceRegistryAmqpServer extends AmqpServiceBase<ServiceConfigProperties> {
 
     @Override
     protected String getServiceName() {
         return Constants.SERVICE_NAME_DEVICE_REGISTRY;
     }
+
 }
```

Comment:
new line

Reply:
I added a new line as the class starts with a empty line.